src/main/java/seedu/address/MainApp.java

```diff
@@ -99,24 +103,31 @@ private Model initModelManager(Storage storage, ReadOnlyUserPrefs userPrefs) {
 
         ReadOnlyIdentifiableManager<Pilot> pilotManager;
         Optional<? extends ReadOnlyIdentifiableManager<Pilot>> pilotManagerOptional;
+        ReadOnlyIdentifiableManager<Location> locationManager;
+        Optional<? extends ReadOnlyIdentifiableManager<Location>> locationManagerOptional;
 
         try {
```

Comment:
It would be better if the errors could be handled individually. However, this would make this part too bloated. I think that in the future, we can refactor this part to make the code simpler. I will create an issue for this.